protosocket-server/src/connection_server.rs

```diff
@@ -21,6 +22,7 @@ pub trait ServerConnector: Unpin {
         optional_outbound: mpsc::Sender<
             <<Self::Bindings as ConnectionBindings>::Serializer as Serializer>::Message,
         >,
+        address: SocketAddr,
```

Comment:
This is fine, but probably your use case is better served with the `protosocket_rpc::server` module. It [already does this](https://github.com/kvc0/protosocket/blob/dc1b6bb3a2c327d620d2af7a46c32c1e1be474eb/protosocket-rpc/src/server/server_traits.rs#L32-L33). ([example](https://github.com/kvc0/protosocket/blob/main/example-proto/src/server.rs#L69-L72))